### PR TITLE
blocksync: complete transition from Blockchain to BlockSync

### PR DESCRIPTION
**Pull request by Wondertan**
The leftover from https://github.com/tendermint/tendermint/pull/6755

### CODE REVIEW

File: node/setup.go [hunk 1]
```diff
@@ -727,7 +727,7 @@ func makeNodeInfo(
 	var bcChannel byte
 	switch config.BlockSync.Version {
 	case cfg.BlockSyncV0:
-		bcChannel = byte(bcv0.BlockchainChannel)
+		bcChannel = byte(bcv0.BlockSyncChannel)
 
 	case cfg.BlockSyncV2:
 		bcChannel = bcv2.BlockchainChannel
```
Comment (cmwaters) [inline node/setup.go:733]: We're not worried about V2 right? 🙃

File: internal/blocksync/v0/reactor.go [hunk 2]
```diff
@@ -299,8 +299,8 @@ func (r *Reactor) handleMessage(chID p2p.ChannelID, envelope p2p.Envelope) (err
 }
 
 // processBlockchainCh initiates a blocking process where we listen for and handle
-// envelopes on the BlockchainChannel and blockchainOutBridgeCh. Any error encountered during
-// message execution will result in a PeerError being sent on the BlockchainChannel.
+// envelopes on the BlockSyncChannel and blockchainOutBridgeCh. Any error encountered during
```
Comment (cmwaters) [inline internal/blocksync/v0/reactor.go:302]: What's blockchainOutBridgeCh? Is that an actual channel

**Review by cmwaters — APPROVED** [review]

**Comment by cmwaters** [comment]
Thanks @Wondertan for following up

File: node/setup.go (hunk 1 above)
Comment (Wondertan) [reply node/setup.go:733]: I saw some discussions about canceling v2 and also #6755 didn't touch upon it, so I decided not to touch it as well.

File: internal/blocksync/v0/reactor.go (hunk 2 above)
Comment (Wondertan) [reply internal/blocksync/v0/reactor.go:302]: Not sure, It was there, so kept it. Can remove it though, if needed.

Comment (Wondertan) [reply internal/blocksync/v0/reactor.go]: Oh nvm, I've updated the `blockchainOutBridgeCh` to `blockSyncOutBridgeCh`

**Comment by Wondertan** [comment]
While exploring the code I found out that there are more things to be renamed, so I've added those as well. I guess now the transition of the name for v0 can be considered finished.

File: node/setup.go (hunk 1 above)
Comment (cmwaters) [reply node/setup.go:733]: Yeah totally makes sense. Let's leave it for now.

**Comment by tessr** [comment]
@Wondertan can you add a changelog entry for this?

**Pull request merged**